[Linux] Add `game` category to instance shortcuts

Leafeling opened the pull request.

Commit:
Add game category to generated Linux shortcuts

Signed-off-by: Hazel <hazel@theopensource.company>

ghost commented:
Tiny change, just needs one more review :P

getchoo commented on launcher/FileSystem.cpp:
```diff
@@ -872,6 +872,8 @@ bool createShortcut(QString destination, QString target, QStringList args, QStri
            << "\n";
     stream << "Type=Application"
            << "\n";
+    stream << "Categories=Game"
```
we should probably use the same categories as our regular [desktop file](https://github.com/PrismLauncher/PrismLauncher/blob/36d2109a1596df5ec20cb2b1828fc25846e7d926/program_info/org.prismlauncher.PrismLauncher.desktop.in#L10)

Leafeling replied:
Changed applied.

Commit:
Use same categories as main desktop entry

Trial97 commented:
please do not forget to sign your commits:https://github.com/PrismLauncher/PrismLauncher/pull/1704/checks?check_run_id=18443870695

Commit:
DCO Remediation Commit for Hazel Hofmann <hazel@theopensource.company>

I, Hazel Hofmann <hazel@theopensource.company>, hereby add my Signed-off-by to this commit: 9ada8d906d522ec92cee58dc016e3d51f33d963d

Signed-off-by: Hazel Hofmann <hazel@theopensource.company>

ghost commented:
Will that work?

Trial97 commented:
yes it worked thanks!
